Headlines/src/ViewAnimator/AnimationType.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AnimationType.swift
+//  Pods-ViewAnimator_Example
+//
+//  Created by Marcos Griselli on 10/5/17.
+//
+
+import UIKit
+
+/// AnimationType available to perform/
+///
+/// - from: Animation with direction and distance.
+/// - zoom: Zoom animation.
+/// - rotate: Rotation animation.
+public enum AnimationType: Animation {
+
+    case from(direction: Direction, offset: CGFloat)
+    case zoom(scale: CGFloat)
+    case rotate(angle: CGFloat)
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewAnimator/AnimationType.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AnimationType.swift:20</code></td>
    </tr>
  </tbody>
</table>